swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
+  "meta": {
+    "autorest_options": {
```

Comment:
Do we need these settings for the new generator?

Reply:
This file will be updated by @PhoenixHe-msft after we merge the script. For now I've removed the extra settings.